features/external_tables.md

```diff
@@ -56,3 +57,101 @@ On many Postgres providers, you can also set up this connection in the other dir
 ## Other External Table Engines
 
 Hydra is working on providing connections to other data sources in the near future. If you have a request, please reach out to us via our support channel.
+
+## MySQL External Tables
+
+MySQL External Tables are implemented using [`mysql_fdw`](https://github.com/EnterpriseDB/mysql_fdw).
+To set up a MySQL External Table from `psql`, run the following SQL, replacing `...` with your server's information.
+
+```sql
+CREATE EXTENSION mysql_fdw;
+
+CREATE SERVER remote_mysql_server FOREIGN DATA WRAPPER mysql_fdw OPTIONS (host '...', port '...');
+
+CREATE USER MAPPING FOR CURRENT_USER SERVER remote_mysql_server OPTIONS (username '...', password '...');
+```
+
+You can now create foreign tables from your remote MySQL database.
+Assuming your remote MySQL database's name is `mysql`, you can [import all of your remote tables](https://www.postgresql.org/docs/current/sql-importforeignschema.html) at once as follows:
+
+```sql
+CREATE SCHEMA remote_mysql;
+
+IMPORT FOREIGN SCHEMA mysql FROM SERVER remote_mysql_server INTO remote_mysql;
+```
+
+## HTTP External Tables
+
+You can call a HTTP web service to obtain data that lives outside Hydra and create a [view](https://www.postgresql.org/docs/current/sql-createview.html) to consume the data like a table.
+HTTP external tables are implemented using [`pgsql-http`](https://github.com/pramsey/pgsql-http).
+
+The following is an example that calls the GitHub organization repo endpoint to get a list of repos in JSON format and defines a view to hold the result:
+
+```sql
+CREATE EXTENSION http;
+
+CREATE VIEW github_repos AS SELECT json_array_elements(content::json) as repo_json
+  FROM http_get('https://api.github.com/orgs/hydrasdb/repos');
+```
+
+You can query the view as if it's a table. Note that Postgres view is read-only.
+
+The following gets all the repo names from the view:
+
+
+```sql
+SELECT repo_json->'name' FROM github_repos;
```

Comment:
hmm, would it be possible to "expand" the json in the view query, to make it more table-like? Maybe you would have to do something like this:

```sql
CREATE VIEW github_repos AS
WITH json AS (
SELECT json_array_elements(content::json) AS repo
  FROM http_get('https://api.github.com/orgs/hydrasdb/repos')
)
SELECT 
  repo->'name' AS name,
  ... -- add other fields
FROM json;
```

Reply:
Good call. Addressed in https://github.com/HydrasDB/docs/pull/7/commits/c4104787558171fcaaaa2d6d052be60eb5a7a19d